### ISSUE TITLE
Remember window size and position

### ISSUE THREAD

**Issue by skylarnagao** [PeerioTechnologies/peerio-client#272]
User request from ES:

If user changes Peerio's window size and position, remember this size/position the next time the app is launched.

**Comment by vdcbb** [comment]
Yes, this is a much needed enhancement. I prefer my windows open maximized. Peerio never does this, and it is very annoying having to maximize the window every single time I open the app.
